canonical-websites/www.ubuntu.com
# Update desktop developers page

## Done

Update copy and images on `/desktop/developers`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/desktop/developers>
- Check that page matches [the design](https://user-images.githubusercontent.com/36884067/38744378-98a2bc3c-3f39-11e8-8aad-c7685a03382a.jpg) and [the copy doc](https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit#)


## Issue / Card

Fixes #2888
